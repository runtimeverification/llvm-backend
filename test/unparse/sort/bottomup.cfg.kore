// RUN: %kprint-nofilter-check
\or{SortGeneratedTopCell{}}(
  \and{SortGeneratedTopCell{}}(
    \and{SortGeneratedTopCell{}}(
      \equals{SortString{}, SortGeneratedTopCell{}}(
        Var'Unds'Gen0:SortString{},
        \dv{SortString{}}("aaa")
      ),
      \equals{SortString{}, SortGeneratedTopCell{}}(
        VarOut:SortString{},
        \dv{SortString{}}("ccc")
      )
    ),
    \equals{SortString{}, SortGeneratedTopCell{}}(
      VarOut:SortString{},
      \dv{SortString{}}("bbb")
    )
  ),
  \and{SortGeneratedTopCell{}}(
    \and{SortGeneratedTopCell{}}(
      \equals{SortString{}, SortGeneratedTopCell{}}(
        Var'Unds'Gen0:SortString{},
        \dv{SortString{}}("bbb")
      ),
      \equals{SortString{}, SortGeneratedTopCell{}}(
        VarOut:SortString{},
        \dv{SortString{}}("ddd")
      )
    ),
    \equals{SortString{}, SortGeneratedTopCell{}}(
      VarOut:SortString{},
      \dv{SortString{}}("aaa")
    )
  )
)
