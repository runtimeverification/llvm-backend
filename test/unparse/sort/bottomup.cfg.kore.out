    {
      Out:String
    #Equals
      "aaa"
    }
  #And
    {
      Out:String
    #Equals
      "ddd"
    }
  #And
    {
      _Gen0:String
    #Equals
      "bbb"
    }
#Or
    {
      Out:String
    #Equals
      "bbb"
    }
  #And
    {
      Out:String
    #Equals
      "ccc"
    }
  #And
    {
      _Gen0:String
    #Equals
      "aaa"
    }
